transport/src/main/java/io/camunda/zeebe/transport/stream/impl/ClientStream.java

```diff
@@ -17,4 +19,22 @@
     AggregatedClientStream<M> serverStream,
     DirectBuffer streamType,
     M metadata,
-    ClientStreamConsumer clientStreamConsumer) {}
+    ClientStreamConsumer clientStreamConsumer) {
+
+  ActorFuture<Void> push(final DirectBuffer payload, final ConcurrencyControl executor) {
+    final ActorFuture<Void> result = executor.createFuture();
+    try {
+      clientStreamConsumer
+          .push(payload)
+          .thenAccept(ignore -> result.complete(null))
+          .exceptionally(
```

Comment:
:wrench: `exceptionally` is used to convert the result of an exception into a success. This isn't really what we want to do here, so I'd rather just use `whenComplete`, e.g.

```java
clientStreamConsumer.push(payload).whenComplete(result);
```

I think `ActorFuture` is a `BiConsumer`, so it should just work.